src/list/ListItem.js

```diff
@@ -222,8 +278,11 @@ styles = StyleSheet.create({
     marginRight: 5,
     color: colors.grey4
   },
-  chevron: {
-  }
+  textInputStyle: {
+    height: 20,
+    textAlign: 'right'
+  },
+  chevron: {}
```

Comment:
@Joshuabaker2 small thing but can you remove unused styles. Rest all looks good 👍